### DISCUSSION TITLE
ETA on a new release being cut?

### DISCUSSION THREAD

**Discussion by stuartbirrell** [nginx-proxy/acme-companion#1150] [Q&A] [answered]
Is there any indications on when there might be a new release getting cut? Last one was Jun 6th.

**Comment by buchdag** [comment, marked as answer]
@stuartbirrell I'll release a new version by the end of the week.

**Reply by buchdag** [reply]
Done ✅

**Reply by stuartbirrell** [reply]
Great news, thanks for that @buchdag
